### PR TITLE
Fix opening a sync realm that is already open elsewhere

### LINKED ISSUE #1593: Can't open a synchronized Realm open in a different process
The Core error message is `History type not consistent` which is not one of the magic strings we check in https://github.com/realm/realm-studio/blob/b3a348bb866ede5a8eda9ac157c79c725294e328/src/ui/reusable/RealmLoadingComponent/index.tsx#L124.

### PR DESCRIPTION
**Pull request by nirinchev**
Fixes https://github.com/realm/realm-studio/issues/1593

### CODE REVIEW

File: RELEASENOTES.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 ### Fixed
 
-- None
+- Fixed an issue where opening a synchronized Realm that is open in another process would result in an error with a message `History type not consistent`. ([#1593](https://github.com/realm/realm-studio/issues/1593))
```
Comment (elle-j) [inline RELEASENOTES.md:9]: Move to CHANGELOG?

**Review by elle-j — APPROVED** [review]

File: RELEASENOTES.md (hunk 1 above)
Comment (nirinchev) [reply RELEASENOTES.md:9]: I think the way Studio works is you add notes to the RELEASENOTES.md which are then moved to CHANGELOG during the release process.

Comment (elle-j) [reply RELEASENOTES.md:9]: Alrighty 🙂 , would have expected it to be the other way around 😛 
(p.s. it had failed `changelog` checks)

Comment (nirinchev) [reply RELEASENOTES.md:9]: Yeah, Studio is the only different one and the check isn't correct for the way the repo is setup.

Comment (elle-j) [reply RELEASENOTES.md:9]: Ah okay I see! 👍

**Pull request merged**